app/src/androidTest/java/io/gnosis/tests/TestApplication.kt

```diff
@@ -32,7 +32,7 @@ open class TestApplication : MultiDexApplication(), ComponentProvider {
 
     companion object Companion {
         fun mockComponent(): ApplicationComponent =
-            (InstrumentationRegistry.getInstrumentation().targetContext.applicationContext as TestApplication).run {
+            (InstrumentationRegistry.getInstrumentation().targetContext.applicationContext as _root_ide_package_.io.gnosis.tests.TestApplication).run {
```

Comment:
`_root_ide_package_` ?

Reply:
just so left over rubbish from refactoring the package name.